### DISCUSSION TITLE
JCenter shutdown

### DISCUSSION THREAD

**Discussion by jarekankowski** [detekt/detekt#3458] [General]
Hey! Do you have any plans for moving away from the dependency on JCenter as [it's sunset on May 1st](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/)? It looks like the current implementation requires is for `org.jetbrains.kotlinx:kotlinx-html`.

**Comment by picklebento** [comment]
Thank you for reminding. Yes we are aware of Bintray/JCenter deprecation and are currently working on it. We hope `kotlinx-html` will be published on mavenCentral to ensure a smoother move.

**Reply by jarekankowski** [reply]
Great, thank you so much and good luck with migration!

**Reply by BraisGabin** [reply]
#3455 to see the progress

**Comment by cortinico** [comment]
For future reference: we're currently blocked by this https://github.com/Kotlin/kotlinx.html/issues/173. Hopefully JetBrains will migrate kotlinx packages and will unblock us. If that it's not the case, we will find another alternative.

**Reply by picklebento** [reply]
There is an update from JFrog:
```
UPDATE: To better support the community in this migration, JFrog has extended the JCenter new package versions submission deadline through March 31st 2021.

To clarify, the JCenter repository will keep serving packages for 12 months until February 1st 2022. Only the JCenter REST API and UI will be sunsetted on May 1st 2021.
```
